### ISSUE TITLE
Record.patch

### ISSUE THREAD

**Issue by ronag** [deepstreamIO/deepstream.io-client-js#146]
Since the path updates use patch I think we should make it accessible, e.g.

```javascript
rec.patch({
  title: 'ASD',
  items: ['FOO']
})
```

Should be the same as

```
rec.set('title', 'ASD')
rec.set('items', ['FOO'])
```

**Comment by WolframHempel** [comment]
We'd like to keep the API as simple as possible and hide most of the inner workings (e.g. patch). To address the issue above we are working on automated deltas ( #83 ) that create patches from `set()` calls without a path

**Comment by yasserf** [comment]
Closing this issue as a won't fix / will fix as in #83
